### DISCUSSION TITLE
[Zur Info] Hintergrundinformationen von KosIT zu BT-146

### DISCUSSION THREAD

**Discussion by dsshad** [LandrixSoftware/XRechnung-for-Delphi#16] [General]
Beim Versuch eine XRechnung mit diesen Eingaben zu generieren:

**Positionen:**
Pos	| Menge	| Einzelpreis	| Rabatt-Satz	| MwSt-Satz	| Gesamt
1		| 2		| 50,00			|					| 7%			| 100,00
2		| 1		| 100,00			| 5%				| 19%			| 95,00

**Gesamtwerte / Zusammenfassung:**
Summe netto: 200,00
abzgl. Rabatt: -5,00
Zwischensumme netto: 195,00
zzgl. MwSt. 19%: 18,05
zzgl. MwSt. 7%: 7,00
Gesamtbetrag brutto: 220,05

Ohne Rabatte "on document level"

Bekam ich beim Validieren der XML Datei im UBL Format einen wenig aussgenden Fehler angezeigt:

"val-sch.2.1 | PEPPOL-EN16931-R120 | error | Pfad: /ubl:Invoice/cac:InvoiceLine[2]"

Habe mir die Infotexte in der Spezifikation zig mal angesehen und kam zu der Erkenntnis, dass entweder der Text zu "PEPPOL-EN16931-R120" oder zu "BT-146" falsch sein muss.

Bei "BT-146" steht, dass es sich um den bereits rabattierten Netto-Preis eines Postens handelt.

Bei der Berechnung, wie sie im Text zu "PEPPOL-EN16931-R120" beschrieben wird, wird vereinfacht gesagt (wenn man von Menge 1 ausgeht) von diesem Preis der Positionsnachlass abgezogen. Das klang für mich wie ein doppelter Abzug vom Rabatt / Nachlass. Beim Eintragen des nicht rabattierten Netto-Preises in "BT-146" war die Rechnung nämlich valide.

Ich habe dann bei KosIT nachgefragt und bekam das als Antwort:

> vielen Dank für den Hinweis. Ich muss Ihnen Recht geben, die Beschreibung von BT-146 ist nicht ganz korrekt. Wie in der Anmerkung aufgeführt bezeichnet BT-146 den Postenpreis abzüglich des Postenpreisrabatts (statt des Rabatts für die Rechnungsposition). Wir werden dies bei der nächsten Überarbeitung der Spezifikation berücksichtigen.
> 
> Zu unterscheiden sind BT-136 „Invoice line allowance amount“ und BT-147 „Item price discount“. Während BT-136 den Abschlagsbetrag auf Ebene der Rechnungszeile/Rechnungsposition bezeichnet, wird mit BT-147 ein Nachlass auf den Artikelpreis/Postenpreis angegeben.
> 
> Sofern Sie also einen Preisnachlass auf Ebene der Rechnungszeile/Rechnungsposition gewähren, ist hierfür BT-136 zu nutzen und in BT-146 der nicht-rabattierte Artikelpreis einzutragen. 

Weiß nicht ob es nützlich ist, aber schadet auch nicht.

Meine Frage wurde im Grunde beantwortet, wobei mir nicht klar ist, wo der Unterschied zwischen
_"Abschlagsbetrag auf Ebene der Rechnungszeile/Rechnungsposition"_
und
_"Nachlass auf den Artikelpreis/Postenpreis"_
sein soll... Der Preis befindet sich ja in der Position und rechnerisch kann es nur ein Ergebnis geben...

**Comment by landrix** [comment]
Ein Listenpreis abzgl. Rabatt ergibt einen Einzelnettopreis ("Nachlass auf den Artikelpreis/Postenpreis").
Auf diesen Einzelnettopreis können dann noch einmal Ab- und Zuschläge gemacht werden ("Abschlagsbetrag auf Ebene der Rechnungszeile/Rechnungsposition")

Ist denn das Problem grundsätzlich gelöst?

**Reply by dsshad** [reply]
Gute Frage. Aktuell trage ich in meinem oben beschriebenen Fall den nicht rabattierten Artikelpreis in "BT-146" ein und die Rechnung wird ohne Fehler validiert.

Allerdings weiß ich nicht, ob das generell richtig ist, und stehe scheinbar auf dem Schlauch, was die beiden Nachlassarten angeht. Laut KoSIT-Antwort (der letzte Absatz) ist der nicht rabattierte Artikelpreis in "BT-146" dann zu verwenden, wenn es sich einen "Preisnachlass auf Ebene der Rechnungszeile/Rechnungsposition" handelt. Die andere Option lautet "Nachlass auf den Artikelpreis/Postenpreis"... Also für mich klingt "Preisnachlass" und "Nachlass auf den Preis" ziemlich gleich. Wobei die erste Option wahrscheinlich wie im vorletzten Absatz "_Abschlagsbetrag_ auf Ebene der Rechnungszeile/Rechnungsposition" heißen sollte und nicht "_Preisnachlass_ auf Ebene der Rechnungszeile/Rechnungsposition". Dann könnte man das vielleicht als zwei unterschiedliche Dinge sehen, sprich als einen festen Abschlag (keinen Abschlag in Prozent, weil es sonst dasselbe in grün wäre).

Ich gebe schon immer den Rabatt-Satz innerhalb der Position an, reduziere den Einzelpreis und rechne dann mit dem reduzierten Einzelpreis weiter, so dass am Ende auch der Gesamtbetrag der Position reduziert ist.

Nach Ansicht von KoSIT wäre das wahrscheinlich eher ein "Nachlass auf den Artikelpreis/Postenpreis" und kein "Abschlagsbetrag auf Ebene der Rechnungszeile/Rechnungsposition" und in dem Fall wäre in "BT-146" der rabattierte Artikelpreis anzugeben, was ich widerum nicht validiert bekomme. **Allerdings nur bei der UBL-Ausgabe nicht. Bei der ZUGFeRD-Ausgabe werden beide Varianten validiert...**

Deswegen bin ich bei dem Thema gerade etwas überfragt und weiß nicht, wo das Problem liegt.
